rest/data/patch.go

```diff
@@ -208,3 +234,29 @@ func (hp *MockPatchConnector) FindPatchesByUser(user string, ts time.Time, limit
 	}
 	return patchesToReturn, nil
 }
+
+func (c *MockPatchConnector) AbortPatchesFromPullRequest(event *github.PullRequestEvent) error {
+	_, _, err := verifyPullRequestEventForAbort(event)
+	return err
+}
+
+func verifyPullRequestEventForAbort(event *github.PullRequestEvent) (string, string, error) {
```

Comment:
it seems like this function is used only once and has a strange interface given it's single place of use.

Reply:
It's used by both implementations of AbortPatchesFromPullRequest.